tests/testthat/test-df_mat_mul.R

```diff
@@ -0,0 +1,19 @@
+df <- data.frame(matrix(1:100, ncol = 5))
+mat <- matrix(1:4, ncol = 2)
+df_mat_mul(df, mat, "z", dplyr::num_range("X", 2:3))
```

Comment:
What is this line doing outside of the test? (The first two create objects available to all the tests)

Reply:
Fixed.